modules/op-resolver-git-override-scm.adoc

```diff
@@ -5,7 +5,7 @@
 :_mod-docs-content-type: REFERENCE
 = Specifying a remote pipeline or task by using the Git resolver with the authenticated SCM API overriding the Git resolver configuration
 
-When creating a pipeline run, you can specify a remote pipeline from a Git repository by using the authenticated SCM API. When creating a pipeline or a task run, you can specify a remote task from a Git repository. You can override the initial configuration settings in specific pipeline runs or tasks to customize the behavior according to different use cases.
+You can override the initial configuration settings in specific pipeline runs or tasks to customize the behavior according to different use cases. You can use this method to access an authenticated provider that is not configured in the `TektonConfig` custom resource (CR).
```

Comment:
I did not get this completely, it is the same as before, only addition is support for multiple git providers

Reply:
Oh yes, this is kind of outside of scope for this PR, it is a line that @mramendi asked me to enhance so that it is clear to users why they might want to do this. It is just an additional change that has nothing to do with multiple git providers :+1:

Reply:
So shall we change/remove it as I think it might confuse the users

Reply:
The change is related to the addition of multiple providers. A user might want to know why we have this feature in addition to the multiple providers feature, which is more neat. This is why I suggested adding the fact that you can use this method to access a provider not in TektonConfig.